pysdsl/types/sorted_int_stack.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <string>
+#include <tuple>
+
+#include <pybind11/pybind11.h>
+
+#include <sdsl/util.hpp>
+#include <sdsl/sorted_int_stack.hpp>
+
+#include "calc.hpp"
+#include "io.hpp"
+#include "operations/iteration.hpp"
+#include "operations/sizes.hpp"
+#include "docstrings.hpp"
+
+
+namespace py = pybind11;
+
+class sorted_int_stack : public sdsl::sorted_int_stack {
```

Comment:
do you create a child class to only override one method? I think using lambdas inside `def()`s would be a better option.